gui/default/syncthing/core/syncthingController.js

```diff
@@ -1097,6 +1097,126 @@ angular.module('syncthing.core')
             }
         };
 
+        $scope.deviceStatusIcon = function(cfg) {
+            switch ($scope.deviceStatus(cfg)) {
+                case 'insync':
+                    return 'fa-check';
+                case 'unused-insync':
+                    return 'fa-unlink';
+                case 'syncing':
+                    return 'fa-sync';
+                case 'paused':
+                    return 'fa-pause';
+                case 'unused-paused':
+                    return 'fa-unlink';
+                case 'disconnected':
+                    return 'fa-power-off';
+                case 'disconnected-inactive':
+                    return 'fa-power-off';
```

Comment:
```suggestion
                case 'disconnected':
                case 'disconnected-inactive':
                    return 'fa-power-off';
```

And similar for `fa-unlink`...

Reply:
Done.